sherpa/astro/tests/test_astro_pha_scale.py

```diff
@@ -1143,3 +1143,269 @@ def test_xspecvar_no_grouping_comparison_xspec(make_data_path,
+                                        1.41421356, 1.        , 0.        , 1.        , 0.        ,
+                                        0.        , 1.        , 0.        , 0.        , 1.        ,
+                                        0.        , 0.        , 0.        , 0.        , 0.        ,
+                                        0.        , 0.        , 0.        , 1.        , 0.        ,
+                                        0.        , 1.        , 1.        , 0.        , 0.        ,
+                                        0.        , 1.        , 1.        , 1.        , 1.        ,
+                                        0.        , 0.        , 0.        , 1.        , 0.        ,
+                                        1.        , 0.        , 0.        , 0.        , 0.        ,
+                                        0.        , 0.        , 1.        , 0.        , 1.        ,
+                                        1.        , 0.        , 1.        , 0.        , 0.        ,
+                                        0.        , 0.        , 1.        , 0.        , 0.        ,
+                                        0.        , 1.        , 0.        , 1.        , 0.        ,
+                                        0.        , 0.        , 0.        , 0.        , 0.        ,
+                                        0.        , 0.        , 0.        , 0.        , 0.        ,
+                                        0.        , 0.        , 0.        , 0.        , 0.        ,
+                                        0.        , 0.        , 0.        , 0.        , 0.        ,
+                                        0.        , 0.        , 0.        , 0.        , 0.        ,
+                                        1.        , 0.        , 0.        , 0.        , 0.        ,
+                                        0.        , 0.        , 0.        , 0.        , 0.        ,
+                                        0.        , 1.        , 0.        , 6.164414  ])
+
+
+# This is the mask to match
+#    notice()
+#    notice(0.5, 2)
+#    notice(5, 7)
+#
+STATERROR_3C273_MASK = np.array([False, False, False,  True,  True,  True,  True,  True,  True,
+                                 True,  True,  True,  True,  True,  True,  True,  True,  True,
+                                 True,  True,  True,  True,  True,  True,  True,  True, False,
+                                 False, False, False, False, False, False, False, False, False,
+                                 False, False, False, False,  True,  True,  True,  True,  True,
+                                 False])
+
+
+# The background dataset now has a different mapping to groups
+# than the source region but still using the "same" energy
+# filter (modulo different buoundaries). The tabStops means
+# that the first 20 and last ~200 channels map to individual
+# groups, hence the large numbre of elements.
```

Comment:
numbre -> number

Reply:
@wmclaugh - you can "suggest" the change which makes it easier for me to apply (all I have to do is apply changes) and you don't need to do a "old -> new" change. See section 6 of https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request but all you need to do is select the +/- icon after the write/preview tabs and it'll fill the text box with the existing text which you can then just chagbe, a la

![github](https://user-images.githubusercontent.com/224638/123424049-500c7c80-d58e-11eb-980c-d8ba6b39e909.png)

Reply:
[I've applied this change but leaving this conversation open for the moment]